src/utilities/dates.ts

```diff
@@ -134,14 +134,6 @@ export function isDateBefore(date: Date, dateToCompare: Date) {
   return date.getTime() < dateToCompare.getTime();
 }
 
-export function isSameDate(source: Date, target: Date) {
```

Comment:
Compare with isSameDay:

https://github.com/Shopify/polaris-react/blob/88868bc44ba464e30827b4a4b49b0f5d89f83f38/src/utilities/dates.ts#L65-L71

I believe this snuck in while I was killing of javascript-utilities - I coped this function from `@shopify/javascript-utilities` without spotting that we already had an equivalent function already in this file